src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx

```diff
@@ -329,8 +333,11 @@ export default function FundingCycleDetails({
           :
         </span>{' '}
         <FundingCycleDetailWarning
-          showWarning={unsafeFundingCycleProperties.ballot}
-          tooltipTitle={riskWarningText.ballot}
+          showWarning={
```

Comment:
maybe we could change `FundingCycleDetailWarning` to just accept a single `{warning: string}` argument where the tooltip is shown if `warning !== undefined` ? could clean up these ternary values here and in FundingAttributes.tsx,

Reply:
Very good call, will make a housekeeping issue for it